### ISSUE TITLE
Profile buttons not saving separately

### ISSUE THREAD

**Issue by thegoldenkrios** [tullamods/Dominos#630]
I've run into this issue on my warrior where I have two separate profiles. One is for Fury spec, and the other is for Protection spec. I have duel spec enabled, and when I switch profiles it treating each profile like they're one together. 
So if I swap to my Fury spec talent set up, and add lets say Bloodthirst to my hotbar on a specific slot(Meanwhile mortal strike in my other spec is in that same slot) But when I switch back to Protection spec it replaces mortal strike with Bloodthirst as if it reconizes the button changes I made in spec 1(profile 1) and puts those changes into spec 2(profile 2). Now the profile shapes, positions of bars are different but the buttons do NOT save. I constantly have to change my buttons around everytime I swap specs regardless of having Duel spec enabled on Dominos with 2 profiles.

I recorded it: https://streamable.com/rrmoit 
Video 2: https://streamable.com/3q1rdr

So basically in this video I am swapping from protection spec to fury spec. I added mortal strike to my 1 key, and mocking blow to my E key. You will see me change specs, and get a brief glimpse of SLAM E on Fury spec, and Bloodthirst 1 on fury spec just before it over rides and places Mortal Strike on 1 and Mocking blow on E.
It doesnt matter what spec or profile it turns to, it doesnt save the buttons. it treats the buttons like theyre one profile on any profile i swap to. Any changes made in Fury profile with buttons are applied to Protection profile, vice versa.
its driving me insane D = 

Fix: Some way to save button lay outs on profiles for duel spec?

World of warcraft version: Classic wotlk latest

**Comment by Tuller** [comment]
Dominos doesn't save what buttons are on your bar - this is controlled on Blizzard's end
